src/main/scala/cromwell/engine/workflow/ValidateActor.scala

```diff
@@ -78,4 +79,55 @@ class ValidateActor(wdlSource: WdlSource, workflowInputs: Option[WdlJson], workf
           APIResponse.fail(ex))
     }
   }
+
+  //TODO: With PBE, this should be defined in the backend.
```

Comment:
Actually, I think it'd be much better later on with a WF level validator for backend, which would be a single method (or a separate class/actor responsible for the validation) (PS: refer the PR that I did on the PBE branch a few days ago)